tracer/dependabot/Datadog.Dependabot.Integrations.csproj

```diff
@@ -138,13 +138,13 @@
 
     <!-- Integration: NUnit3.TestAdapter -->
     <!-- Assembly: NUnit3.TestAdapter -->
-    <!-- Latest package https://www.nuget.org/packages/NUnit3TestAdapter/4.1.0 -->
-    <PackageReference Include="NUnit3TestAdapter" Version="4.1.0" />
+    <!-- Latest package https://www.nuget.org/packages/NUnit3TestAdapter/4.2.0 -->
+    <PackageReference Include="NUnit3TestAdapter" Version="4.2.0" />
 
     <!-- Integration: Oracle.ManagedDataAccess -->
     <!-- Assembly: Oracle.ManagedDataAccess -->
-    <!-- Latest package https://www.nuget.org/packages/Oracle.ManagedDataAccess/21.4.0 -->
-    <PackageReference Include="Oracle.ManagedDataAccess" Version="21.4.0" />
+    <!-- Latest package https://www.nuget.org/packages/Oracle.ManagedDataAccess/21.5.0 -->
+    <PackageReference Include="Oracle.ManagedDataAccess" Version="21.5.0" />
```

Comment:
It also looks like this package update isn't reflected in the package versions generated files. I think we need to add an entry for Samples.OracleMDA which uses this NuGet package, which can be done in a separate PR

Reply:
Yeah, we don't currently run the Oracle tests at all (because, Oracle), so this is all kinda moot. Tbh, I'd rather we don't add the entry until we actually figure out how to run the tests, so we don't get a (bigger) false sense of security